### DISCUSSION TITLE
Download Issue

### DISCUSSION THREAD

**Discussion by MGcrazen** [lllyasviel/Fooocus#3287] [Q&A] [answered]
I've been trying to download the content, but I'm encountering an issue where the process gets stuck on the as the pic given, It doesn't progress beyond this point, and I'm not sure what might be causing the problem, and i already download the file manually and put it into checkpoints directory .
![image](https://github.com/user-attachments/assets/d404a4cd-df84-401a-90f5-a61b9ebd08e4)
![image](https://github.com/user-attachments/assets/58b107dc-b729-4905-ac8b-f2711d5526e3)

**Comment by mashb1t** [comment, marked as answer]
Restarting should recognize the file as downloaded, which solves the issue.

**Reply by MGcrazen** [reply]
Thank you so much men <3

**Comment by ILikeToasters** [comment]
You are going to want to rename that filename to remove the (1) part.

**Reply by MGcrazen** [reply]
Thanks dude!
